internal/pkg/service/stream/storage/level/local/encoding/encoder/csv/csv.go

```diff
@@ -2,30 +2,55 @@ package csv
 
 import (
 	"io"
+	"sync"
 
+	"github.com/keboola/keboola-as-code/internal/pkg/service/stream/mapping/recordctx"
 	"github.com/keboola/keboola-as-code/internal/pkg/service/stream/mapping/table/column"
 	"github.com/keboola/keboola-as-code/internal/pkg/service/stream/storage/level/local/encoding/encoder/csv/fastcsv"
 	"github.com/keboola/keboola-as-code/internal/pkg/service/stream/storage/model"
 	"github.com/keboola/keboola-as-code/internal/pkg/utils/errors"
 )
 
 type Encoder struct {
-	columns column.Columns
-	pool    *fastcsv.WritersPool
+	columns     column.Columns
+	writersPool *fastcsv.WritersPool
+	valuesPool  *sync.Pool
 }
 
+var columnRenderer = column.NewRenderer() //nolint:gochecknoglobals // contains Jsonnet VMs sync.Pool
```

Comment:
:man_shrugging: practically there is currently no difference

If the sync.Pool wasn't there, I'll create a new renderer for each encoder.

I would put it in dependencies if I needed to use dummy implementation in some tests, but it doesn't look like it would be necessary.